gammasky/catalogs.py

```diff
@@ -10,50 +10,92 @@
     'make_tev_catalog_data',
     'make_3fgl_catalog_data',
     'make_2fhl_catalog_data',
-    'make_snrcat_catalog_data',
+    'make_snrcat_catalog_data'
 ]
 
 TO_JSON_KWARGS = dict(orient='split', double_precision=5)
 
 
 def make_tev_catalog_data(nrows=None):
-    # TODO: change to gamma-cat
-    click.secho('Skipping TeV catalog ... need to switch to gamma-cat', fg='red')
-    return
-
-    click.secho('Making TeV catalog data...', fg='green')
+    click.secho('Making TeV catalog data (from gamma-cat)...', fg='green')
 
     out_dir = Path('src/app/data/cat')
     out_dir.mkdir(parents=True, exist_ok=True)
 
-    url = 'https://github.com/gammapy/gamma-cat/blob/master/other_cats/tgevcat/tgevcat.ecsv?raw=true'
-    table = Table.read(url, format='ascii.ecsv')
-    cols = table.colnames
+    url = 'https://github.com/gammapy/gamma-cat/raw/master/docs/data/gammacat.fits.gz'
+    table = Table.read(url)
+    # gamma-cat already has a source_id column
+    cols = [
+        'source_id',
+        'common_name',
+        'gamma_names',
+        'other_names',
+        'ra',
+        'dec',
+        'glon',
+        'glat',
+    ]
+    # cols = table.colnames  <-- Not working right now because I get the error: Cannot convert a table with multi-dimensional columns to a pandas DataFrame.
```

Comment:
You could also read https://github.com/astropy/astropy/issues/4604#issuecomment-184425733 to get some info on Astropy table, pandas Dataframe and JSON serialisation.

Short answer: to drop all multi-dimensional columns you can use this code snippet to make a list of columns to keep, and then index into the table with that list of colnames:
```python
cols = tuple(name for name in table.colnames if len(table[name].shape) <= 1)
t2 = table[cols]
```